app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -137,7 +139,9 @@ class HomeScreenRobot {
 
     fun verifyExistingTopSitesList() = assertExistingTopSitesList()
     fun verifyNotExistingTopSitesList(title: String) = assertNotExistingTopSitesList(title)
+    fun verifyNotExistingSponsoredTopSitesList() = assertNotExistingSponsoredTopSitesList()
```

Comment:
@AaronMT This is the format we've been working with for a long time, started by others, I was thinking if it still makes sense to create 2 functions for one thing. This will probably unnecessarily increase the size of a class.

Reply:
It's not the same thing, the sponsored top sites will have different titles (can't know what websites will be featured as sponsored)

The verification of the sponsored top sites is made based on the subtitle "Sponsored" and the parent index (be sure they are displayed on the second and third row)

Reply:
Yeap, I wasn't making myself clear enough here, sorry @AndiAJ. I was referring to verifyNotExistingSponsoredTopSitesList and assertNotExistingSponsoredTopSitesList - if they should be 2 different functions, or we could refactor and just go with one of them. I have seen large robot classes where refactoring this style of declaring methods would decrease their length by a lot.

Reply:
I think it's fine for the purposes of this test for now, but @sv-ohorvath I'm 100% open to smart refactoring of our robots to decrease their size where it makes sense. Can you make an issue on on our board that gives some of the details?

Reply:
Thanks guys, left it as it is for now.